docker-compose.integration.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  wdqs:
+    image: wikibase/wdqs:0.3.40
```

Comment:
I wonder if this should infact use the wbstack build image from the query service repo?